setup_dcpandas.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Install dcpandas."""
+from setuptools import setup, find_packages
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+# Package metadata.
+NAME = 'dcpandas'
+DESCRIPTION = 'A library to create pandas objects using the Data Commons Python API.'
+URL = 'https://github.com/datacommonsorg/api-python'
+EMAIL = 'support@datacommons.org'
+AUTHOR = 'datacommons.org'
+REQUIRES_PYTHON = '>=2.7'
+VERSION = '0.0.1'
+
+REQUIRED = [
+    'six',
+    'pandas',
+]
+
+PACKAGES = ['dcpandas']
```

Comment:
might rename this to datacommons-pandas as well

Reply:
Yeah, let me do that last to preserve comments.